lib/index.js

```diff
@@ -346,8 +344,8 @@ api.get = async ({
   }
   // filter expired token.
   if(filterExpiredTokens) {
-    const now = Date.now();
-    if(Date.parse(rValue.expires) < now) {
+    const now = new Date(Date.now());
```

Comment:
Isn't this the same as `new Date()`?

Reply:
yep it is. I wasn't sure how to stub the constructor in test, so  I stubbed Date.now() there.

Reply:
I see.  A sinon issue.  I imagine future readers may assume these calls can be optimized.  Would be better if plain `new Date()` would work.  Did you look into the fake timers API?

Reply:
@davidlehn Thanks,  that worked.

Reply:
@davidlehn Do you think I should remove the `else if` block here https://github.com/digitalbazaar/bedrock-authn-token/pull/33/files#diff-6d186b954a58d5bb740f73d84fe39073R554. Currently only nonce type has expiration date set and rest of the types don't. So I think this code is not useful at the moment.

Reply:
@JSAssassin If no other types have expires, and we don't plan on that in the immediate future, then [sure](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).  Might be good to leave a comment noting that is where to add back expires checking if needed.